.travis.yml

```diff
@@ -6,6 +6,7 @@ php:
   - 5.6
   - 7.0
   - 7.1
+  - 7.2
```

Comment:
testing on 5.6 and 7.0 seems unnecessary in our situation.

Reply:
Updated, but because travis isn't enabled on this repo, it doesn't matter much ;)